File: LeoApp/app/src/main/java/de/slg/leoapp/sqlite/Klausur.java
package de.slg.leoapp.sqlite;

import java.text.SimpleDateFormat;
import java.util.Date;

public class Klausur {
    private final int    id;
    private final String titel;
    private final Date   datum;
    private final String notiz;

    public Klausur(int id, String titel, Date datum, String notiz) {
        this.id = id;
        this.titel = titel;
        this.datum = datum;
        this.notiz = notiz;
    }

    public String getTitel() {
        return titel;
    }

    public Date getDatum() {
        return datum;
    }

    public String getNotiz() {
        return notiz;
    }

    public int getId() {
        return id;
    }

    @Override
    public String toString() {
        return getId() + "," + getTitel() + ',' + new SimpleDateFormat("dd.MM.yyyy").format(getDatum()) + ',' + getNotiz();
    }
}

File: exams/src/main/java/de/slg/leoapp/exams/Klausur.kt
package de.slg.leoapp.exams

import de.slg.leoapp.exams.data.db.Subject
import java.util.*

/**
 * @author Moritz
 * Erstelldatum: 03.09.2018
 */
class Klausur(
        val id: Int?,
        val subject: Subject,
        val datum: Date
)